Sponsors.md

```diff
@@ -1,90 +1,91 @@
+* "Thank you" message by organizers during Meetup (not by company),
+* Sponsor's slide in DevStaff Welcome presentation,
+* "Thank you" mention in our social media.
+
+### Venue Sponsor
 
-## Venue Sponsor
 High quality human interactions happen more naturally in a nice setting. The venue is one of the most important factors that go into running successful meet-ups. 
 
-We want venues that are accessible, large enough for our size and provide the necessary equipment (e.g. fast internet connection, projector, comfortable seating, etc.) to help our meet-ups run smoothly. You can find more details in [this relevant discussion](https://github.com/devstaff-crete/DevStaff-Heraklion/issues/185). 
+We want venues that are accessible, large enough for our size and provide the necessary equipment (e.g. fast internet connection, projector, comfortable seating, etc.) to help our meet-ups run smoothly. You can find more details in [this relevant discussion](https://github.com/devstaff-crete/DevStaff-Heraklion/issues/185). We prefer a venue that is free and without any unreasonable demands. 
 
 The list below describes the awareness benefits that our venue sponsor receives in return:
-* leaflets (can be left on table at entrance),
+* leaflets (can be left on a table at entrance),
 * online banner on DevStaff website (with back-link do-follow),
-* “Thank-you” message by organizers during Meetup (not by company),
+* "Thank-you" message by organizers during Meetup (not by company),
 * Sponsor’s slide in DevStaff Welcome presentation,
-* “Thank you” mention in facebook page post.
+* "Thank you" mention in our social media.
 
-## How to contact us about Sponsorship opportunities
+### How to contact us about Sponsorship opportunities
 Depending on the time of year and ongoing sponsor engagements, some, or all, of the above options may or may not be available. Please contact us at info @t devstaff d0t gr, so you can get in touch with the Org Team and discuss the available options. 
 
-## History 
-After several Org Team Meetups on the topic, we reached a consensus that we will only open up a Catering sponsorship, 
-so that we can have some snacks & drinks for every meetup. 
-
-In addition, after the discussion in [this issue](https://github.com/devstaff-crete/DevStaff-Heraklion/issues/185) and several org meetups, we also opened the possibility of a Venue Sponsor, as long as they can provide their venue for free and without any unreasonable demands. 
-
 ### Sponsor Eligibility
-We did discuss about whether this sponsorship should be open to IT companies at all (so that we wouldn't face any competition / 
-conflict of interest issues), however, we agreed that we will not place any explicit constraints for the time being. 
+We did discuss about whether sponsorship should be open to IT companies at all (so that we wouldn't face any competition / conflict of interest issues), however, we agreed that we will not place any explicit constraints for the time being. 
```

Comment:
I made this as a sub-section of "Other ways to sponsor", as I am not sure we can vet OpenCollective sponsors (anyone can contribute there, right?).